### PR TITLE
Plugin docs

### PR DESCRIPTION
**Pull request by markstory**
Explain that bake events can be bound in plugins. And try to clarify the plugin chapter a bit.

### CODE REVIEW

**Comment by cake17** [comment]
@markstory I added the fr translation

File: en/bake/development.rst [hunk 1]
```diff
@@ -33,7 +33,10 @@ be used::
 
     });
 
-Bake events can also be useful for making small changes to existing templates.
+If you want to modify bake from within another plugin. Putting your plugin's
```
Comment (antograssiot) [inline en/bake/development.rst]: is it just me or does it make more sense with a coma than a dot ?

```
If you want to modify bake from within another plugin, putting you

File: fr/bake/development.rst [hunk 2]
```diff
@@ -34,12 +34,15 @@ un autre helper à la classe de vue bake, cet event peut être utilisé::
...
     });
 
-Les events de Bake peuvent aussi être utiles pour faire des petits
-changements aux templates existants. Par exemple, pour changer les noms de
-variable utilisés lors de la création avec bake de fichiers de
-controller/template, on pourra utiliser une fonction qui écoute
-``Bake.beforeRender`` pour modifier les variables utilisées dans les templates
-de bake::
+Si vous souhaitez modifier bake à partir d'un autre plugin. Mettre vos events
+de bake du plugin dans le fichier ``config/bootstrap.php`` du plugin est une
```
Comment (antograssiot) [inline fr/bake/development.rst]: ``Mettre les events de bake de votre plugin`` peut être

File: fr/bake/development.rst [hunk 3]
```diff
@@ -34,12 +34,15 @@ un autre helper à la classe de vue bake, cet event peut être utilisé::
...
+bonne idée.
+
+Les events de bake peuvent être pratique pour faire de petits changements dans
```
Comment (antograssiot) [inline fr/bake/development.rst]: pratiques ?

**Comment by antograssiot** [comment]
nice work from both of you @markstory @cake17

File: en/bake/development.rst (hunk 1 above)
Comment (HavokInspiration) [inline en/bake/development.rst]: Agreed

**Comment by markstory** [comment]
Thanks @antograssiot and @cake17 !

**Comment by cake17** [comment]
@antograssiot thanks for the correction @markstory thanks for all your work on CakePHP

**Pull request merged**